### ISSUE TITLE
Please create a tag for the 0.2.16 version (CocoaPods)

### ISSUE THREAD

**Issue by kmpnz** [matthewcheok/Realm-JSON#94]
The version was bumped in the `Realm+JSON.podspec` file already, but cause the tag is missing CocoaPods doesn't picked it up and shows the old versio

**Comment by viktorasl** [comment]
Where does Cocoapods show old version?

**Comment by kmpnz** [comment]
@viktorasl Here https://cocoapods.org/?q=Realm%2BJSON

**Comment by minsoe** [comment]
yes, because of that. Realm version is still 0.103.2, not able to use 1.0. 
Showing this in 'pod update' 
- `Realm (~> 1.0)` required by `Podfile`
- `Realm (~> 0.85.0)` required by `Realm+JSON (0.2.0)`

**Comment by ChristianRonningen** [comment]
Workaround for this is to point the pod to a specific commit like so:
`pod 'Realm+JSON', :git => 'https://github.com/matthewcheok/Realm-JSON', :commit => 'e5d69f0fb0d541cc2235a99c70a00b5bd0c0c59c'` when the e5d.... is the latests commit right now, this will now work with Realm 1.0

**Comment by justinfriberg** [comment]
Is this project still being maintained?  Does anyone know why the updated podspec hasn't been submitted to CocoaPods?  Is forking our only option at this point?

We can't use the workaround posted above since we're creating our own pod, and podspecs don't allow git branches.

**Comment by komocode** [comment]
Seems dead it looks like...

**Comment by komocode** [comment]
Ok so I'm not going to pretend how to do this. Author just made me a collaborator. I've never published anything on Cocoapods before so if anyone wants to write up a dumb-proof tutorial so I don't screw this up for everyone, that would be great.
